apps/desktop/plans/pty-handoff-experiment/nodepty-test/counter-handoff-child.js

```
@@ -0,0 +1,72 @@
+    lines: 0,
+    lastSeq: -1,
+    gaps: [],
+    partial: "",
+  };
+  const rs = fs.createReadStream(null, { fd: 3 + i, autoClose: false });
+  const sess = sessions[i];
+  rs.on("data", (chunk) => {
+    sess.bytes += chunk.length;
+    sess.partial += chunk.toString("utf8");
+    let idx;
+    while ((idx = sess.partial.indexOf("\n")) >= 0) {
```

Biome (2.4.13)

[error] The assignment should not be in an expression.

(lint/suspicious/noAssignInExpressions)